metatron-app/metatron-discovery
# #3837 Added password change function in user management page

### Description
Added password change function in user management page

**Related Issue** : #3837 


### How Has This Been Tested?
1. Enter the user detail screen of the user management menu.
2. Check if the Change Password button is visible at the top.
3. When the change button is clicked, a change pop-up is displayed and it is checked if it is modified.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
